cve_bin_tool/extractor.py

```diff
@@ -6,6 +6,7 @@
 """
 import itertools
 import os
+import pathlib
```

Comment:
I'm torn, because pathlib's been around since python 3.4 so it shouldn't be a big deal to add it now that we support only 3.7-3.10. (I think we still supported 2.7 when this extractor code was written, so that's why it didn't use it.)  But on the other hand, it feels so darned weird to import a library then use it only on a few lines in one function. when we could probably just continue to use `os.path.join` for this particular PR.

I think what we should do is change this PR to use the "old" methods and take out this import so it fits with the existing code.

BUT I'm not adverse to having a bigger refactoring PR where you use pathlib throughout extractor (and possibly in the tests that handle file paths?).  I just don't want this to wind up being like we did with requests where part of the code uses it and part does not for a really long time.

Reply:
And if you're not up for a larger refactoring PR now, you can open an issue so someone else can look at it as an option later.

Reply:
noted!! i will work on that after this PR.